### PR TITLE
Add citcom project description

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

File: _project_descriptions/citcom.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 key: citcom
 ---
 
-Project support around use of git and GitHub, and general good software engineering practise, with likely future development on containerised models. 
+Statistical and machine learning approaches are able to make excellent predictions of an output based on several pieces of input data. However, the extent to which each of the inputs contribute to causing the output is generally unclear. Causal inference is a method that seeks to quantify causal relationships between inputs and outputs by testing a "causal graph" describing likely relationships, informed by someone expert in the data. This project, [CITCoM](https://gow.epsrc.ukri.org/NGBOViewGrant.aspx?GrantRef=EP/T030526/1), will democratise access to causal inference, bringing this powerful technique to a broader range of researchers in academia, government and the public sector.
 
-[Details at UKRI](https://gow.epsrc.ukri.org/NGBOViewGrant.aspx?GrantRef=EP/T030526/1).
+This project is in its early stages, and has no outputs at present. It is anticipated that RSE will help with version control of code, good practice in python development, software testing and deployment.
```
Comment (willfurnass) [inline _project_descriptions/citcom.md]: Could add that might use DAFNI.

File: _project_descriptions/citcom.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 key: citcom
 ---
 
-Project support around use of git and GitHub, and general good software engineering practise, with likely future development on containerised models. 
+Statistical and machine learning approaches are able to make excellent predictions of an output based on several pieces of input data. However, the extent to which each of the inputs contribute to causing the output is generally unclear. Causal inference is a method that seeks to quantify causal relationships between inputs and outputs by testing a "causal graph" describing likely relationships, informed by someone expert in the data. This project, [CITCoM](https://gow.epsrc.ukri.org/NGBOViewGrant.aspx?GrantRef=EP/T030526/1), will democratise access to causal inference, bringing this powerful technique to a broader range of researchers in academia, government and the public sector.
```
Comment (willfurnass) [inline _project_descriptions/citcom.md]: Matches with my (basic) understanding!

File: _project_descriptions/citcom.md [hunk 3]
```diff
@@ -2,6 +2,8 @@
...
 
-[Details at UKRI](https://gow.epsrc.ukri.org/NGBOViewGrant.aspx?GrantRef=EP/T030526/1).
+This project is in its early stages, and has no outputs at present. It is anticipated that RSE will help with version control of code, good practice in python development, software testing and deployment.
+
+Understanding causality in predictive models is key to understanding which of the inputs are leading to changes in outputs. This is essential to anyone wishing to enact a policy to change the output in future. In the private sector, this might mean understanding what features of a web page (inputs) make it more likely for a customer to buy a product (output). In the public sector, it might mean understanding which public health interventions lead to behavioral change promoting better health.
```
Comment (willfurnass) [inline _project_descriptions/citcom.md:9]: Could also note that helps with testing: (I think) one can determine which variables to control for when looking at the effects of other variables.

**Review by willfurnass — APPROVED** [review]
Minor suggestions but otherwise LGTM.

File: _project_descriptions/citcom.md (hunk 1 above)
Comment (bobturneruk) [reply _project_descriptions/citcom.md]: Done.

File: _project_descriptions/citcom.md (hunk 2 above)
Comment (bobturneruk) [reply _project_descriptions/citcom.md]: 👍

File: _project_descriptions/citcom.md (hunk 3 above)
Comment (bobturneruk) [reply _project_descriptions/citcom.md:9]: I'm going to leave this for now, thanks. I'm sure I'll come back to this as I get more involved with the project.

**Review by willfurnass — APPROVED** [review]

**Pull request merged**